itez/beneficiary/migrations/0001_initial.py

```diff
@@ -15,8 +16,32 @@ class Migration(migrations.Migration):
             name='Province',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('name', models.CharField(blank=True, max_length=255, null=True, verbose_name='Province')),
+                ('name', models.CharField(max_length=255, verbose_name='Province')),
                 ('created', models.DateTimeField(auto_now_add=True)),
             ],
+            options={
+                'verbose_name': 'Province',
+                'verbose_name_plural': 'Provinces',
+                'ordering': ['-created'],
+            },
+        ),
+        migrations.CreateModel(
+            name='WorkDetail',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('salary', models.IntegerField(verbose_name='Monthly Salary')),
+                ('company', models.CharField(max_length=200, verbose_name='Company Name')),
+                ('insured', models.CharField(choices=[('Yes', 1), ('No', 2)], default='Yes', max_length=100, verbose_name='Is your company insured?')),
+                ('work_address', models.CharField(max_length=500, verbose_name='Work Address')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='District',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, verbose_name='District')),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('province', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='beneficiary.province')),
+            ],
         ),
```

Comment:
This is wrong @chriford 
you can not delete migrations and restart from initial no

you must be able to build on existing migrations and it will increment from there

fix the migrations such that you dont delete existing ones
delete and pull from upstream to inherit existing migrations and them do a fresh makemigrations locally  then migrate

Reply:
Alright, let me do it quickly

Reply:
something prompted me to do that though